src/Impostor.Server/Net/Inner/Objects/Systems/ShipStatus/ReactorSystem.cs

```diff
@@ -45,8 +45,12 @@ public async ValueTask Deserialize(IMessageReader reader, bool initialState)
                 for (var i = 0; i < count; i++)
                 {
                     var playerId = reader.ReadByte();
```

Comment:
Issue with the **count** being used to read bytes for PlayerIDs.

Reply:
The deserialization was already implemented in the dev branch. I think the problem was that I was reading the console id only if the player is not null, skipping a byte sometimes.

I can't reproduce the player being null. The only instance I found is when the round starts but that should be covered by initial state check.

Reply:
Ahh, I get it now. Such a simple bug in the code I didn't see!
It was read in twice, by accident, causing an where the 2nd entry (even if they are there) being skipped.
This could had caused the null player issue as well, but for safe measure, I think I'll keep it in.
Thanks for the quick fix!